deployments/mumbai/usdc/configuration.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenPriceFeed": "0x572dDec9087154dC5dfBB1546Bb62713147e0Ab0",
+  "borrowMin": "1000e6",
+  "storeFrontPriceFactor": 0.5,
+  "targetReserves": "5000000e6",
+  "rates": {
+    "supplyKink": 0.8,
+    "supplySlopeLow": 0.0325,
+    "supplySlopeHigh": 0.4,
+    "supplyBase": 0,
+    "borrowKink": 0.8,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 0.25,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "0.000011574074074074073e15",
+    "baseBorrowSpeed": "0.0011458333333333333e15",
+    "baseMinForRewards": "1000000e6"
+  },
+  "assets": {
+    "SAND": {
```

Comment:
we use `asset0` in a number of scenarios, so the first listed asset needs to be something that we can easily mint a large number of. i've never heard of SAND before, but it's very easy to mint

Reply:
lol interesting that they support SAND but not LINK on mumbai

Reply:
weirdly, they have a `LINK / MATIC` feed, but not `LINK / USD` feed

Reply:
🤔